packages/nimble/R/cppDefs_ADtools.R

```diff
@@ -349,7 +338,7 @@ makeADargumentTransferFunction <- function(newFunName = 'arguments2cppad', targe
 ## The code this generates is embedded in the ADtapingFunction made by makeADtapingFunction
 ##
 ## Note this does some work similar to BUGScontextClass::embedCodeInForLoop
-makeCopyingCodeBlock <- function(LHSvar, RHSvar, indexList, indicesRHS = TRUE, incrementIndex, isNode) {
+makeCopyingCodeBlock <- function(LHSvar, RHSvar, indexList, indicesRHS = TRUE, incrementIndex, isNode = FALSE) {
```

Comment:
aha, I bet this would fix some of the errors I've seen in the `gradients` branch.